Dockerfile

```diff
@@ -1,19 +1,26 @@
-# Start from the latest golang base image
-FROM golang:latest
+FROM golang:1.17.8-bullseye
 
 # Add Maintainer Info
-LABEL maintainer "Alon Greyber <agreyber@covar.com>"
 LABEL org.opencontainers.image.source https://github.com/NCATS-Gamma/robokache
 
 # Set the Current Working Directory inside the container
 WORKDIR /app
 
+# make sure all is writeable for the nru USER later on
+RUN chmod -R 777 .
+RUN mkdir /home/nru && \
+    chmod -R 777 /home/nru
+
```

Comment:
I think we need to make the ./data directory and chmod so it is accessible to nru